packages/server/graphql/public/mutations/addFeatureFlagToOrg.ts

```diff
@@ -0,0 +1,34 @@
+import getRethink from '../../../database/rethinkDriver'
+import {getUserId, isSuperUser} from '../../../utils/authorization'
+import standardError from '../../../utils/standardError'
+import {MutationResolvers} from '../resolverTypes'
+
+const addFeatureFlagToOrg: MutationResolvers['addFeatureFlagToOrg'] = async (
```

Comment:
+1 now that we have this mutation, I think we should rename the other one `addUserFeatureFlag` or `addFeatureFlagToUser` to clarify the difference

Reply:
@nickoferrall In case of renaming I would need to update various documentation, and people should update their saved queries. Is it ok to introduce that breaking change?